packages/zowe-explorer-ftp-extension/README.md

```diff
@@ -12,6 +12,22 @@ Enhancements:
 
 For more information, see [Changelog](https://github.com/zowe/vscode-extension-for-zowe/blob/master/packages/zowe-explorer-ftp-extension/CHANGELOG.md).
 
+## Prerequisites
+
+Host-side prerequisites:
+
+- Check to ensure that z/OS FTP service can be accessed remotely:
+  - Connect to z/OS with the FTP client and run the command `rstat` in the FTP client to check that the option `JESINTERFACELevel` is set to `2`.
+
+Client-side prerequisites:
+
+1. Install [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html) on your local machine.
+1. Install [z/OS FTP Plug-in for Zowe CLI](https://github.com/zowe/zowe-cli-ftp-plugin) on your local machine by using the following command:
+
+   ```bash
+   zowe plugins install @zowe/zos-ftp-for-zowe-cli@latest
```

Comment:
I would remove this command for consistency with the way you arranged the first prerequisite (Install Zowe CLI ). If we provide a procedure in the Prerequisites section, let's be consistent and provide procedures for prerequisites that we list, or let's remove all the procedures and leave the links to the documents that provide instruction on how to meet the prerequisites :)

Reply:
Done. The install instructions for Zowe CLI itself are a bit lengthy, so I would rather go the route of just removing the procedures and replacing them with external links to the same information.